touchforms/formplayer/migrations/0001_initial.py

```diff
@@ -22,7 +22,7 @@ class Migration(migrations.Migration):
                 ('session_name', models.CharField(max_length=100)),
                 ('created_date', models.DateTimeField(default=datetime.datetime.utcnow)),
                 ('last_activity_date', models.DateTimeField(null=True)),
-                ('user', models.ForeignKey(to=settings.AUTH_USER_MODEL)),
+                ('user', models.ForeignKey(to=settings.AUTH_USER_MODEL, on_delete=models.CASCADE)),
```

Comment:
Not sure about the behaviour of passing `on_delete` kwarg, but if it's just to get rid of warnings it might be worth to add a comment here to say so (so that it can reviewed when upgrading to Django 2.0 latter)?